package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "jquery": "^2.1.4",
```

Comment:
the branch leaflet-master doesn't exist, so this was erroring. Pulling master instead